docs/keycloak-jwt-authentication-setup.md

```diff
@@ -441,10 +441,10 @@ oc get keycloak -n keycloak
 # Get Keycloak URL
 oc get keycloak keycloak -n keycloak -o jsonpath='{.status.hostname}'
 
-# Get admin credentials (if using the operator-created secret)
-oc get secret credential-keycloak -n keycloak \
+# Get admin credentials (auto-generated by RHBK operator)
+oc get secret rhsso-keycloak-initial-admin -n keycloak \
   -o jsonpath='{.data.username}' | base64 -d
-oc get secret credential-keycloak -n keycloak \
+oc get secret rhsso-keycloak-initial-admin -n keycloak \
```

Comment:
if we are migrating from rhsso to rhbk, why using rhsso?

Reply:
that's created by the operator I think. I only retrieve it. I'll double check

Reply:
Updated to remove the 'rhsso' prefix